### ISSUE TITLE
Clicking once on something from UI panel registers the action twice

### ISSUE THREAD

**Issue by JinxJynx** [WarEmu/WarBugs#20357]
Clicking on something from UI panel once with LMB registers the action twice and it looks as if you clicked twice on that item. It occurs when clicking on Character (P), Backpack (B), Abilities (V) or in Character panel when trying to hide/show an item (cloak for example).

**Expected behavior and actual behavior:**

**Expected behavior:** Clicking once with LMB should register the action only once and not double click on an item.
**Actual behavior:** Clicking once with LMB on the mentioned locations causes the game to register the click twice.

**Steps to reproduce the problem:**
1. Boot Return of Reckoning game client.
2. Login into the Martyrs Square server with a character.
3. Reach the gameplay section of Return of Reckoning and press Alt+Tab to minimize the client.
4. Return into the game client and press on one of the UI items mentioned in bug description (Character (P), Backpack (B) Abilities (V) or 
5. Press P to open the Character panel and attempt to show/hide an item (cloak) by pressing only once on the hide/show tick.
6. Observe that when clicking once in the mentioned locations, the game actually registers the action twice and it appears as if you clicked twice.

**Note:**
1. This issue has a 30% repro rate (3 out of 10 times) and it seems to be triggered after Alt Tabbing and returning into the game client afterwards. I do not currently have any problems with my mouse, tested in other games too.
2. This issue occurs also on any other clickable stuff (abilities, potions, etc), if you attempt to drag and drop them into another place in ability bars, it will use said ability/potion because of the double click instead of dragging it. It also creates issues with equipping talismans into gear.
3. Reloading the interface by using the /rel command did not fix the issue.
4. Addons used: Enemy Addon.

**Testing Screenshots/Videos/Evidences (always needed):**

https://user-images.githubusercontent.com/111609782/185704079-d12ed718-e1ca-40af-a893-73d20d7882ed.mp4

**Comment by xpander69** [comment]
Can't reproduce here. Have you tried without addons and clean character profile settings?
Does this happen with fullscreen/windowed/without borders?
Are you maybe using some thirdparty overlay's like reshade overlay, FPS counters etc that may cause it?

**Comment by lilimx** [comment]
Hey,

I am closing this ticket now as we have not received any further information regarding the problem.
If you are still the opinion that the issue still persists, please open a new ticket with proper evidences.
